src/app/content/highlights/components/__snapshots__/EditCard.spec.tsx.snap

```diff
@@ -138,16 +138,6 @@ exports[`EditCard matches snapshot when editing 1`] = `
       Cancel
     </button>
   </div>
-  <div
-    confirmLink="/accounts/login?r=/"
-    confirmMessage="i18n:highlighting:login:link"
-    data-analytics-label="login"
-    data-analytics-region="highlighting-login"
-    drawFocus={false}
-    message="i18n:highlighting:login:prompt"
-    mock-confirmation={true}
-    onCancel={[MockFunction]}
-  />
```

Comment:
why was login removed from this snap and not the others?

Reply:
This is the result of splitting the ActiveEditCard and the LoginConfirmation. You only get one or the other.